src/polymer/analyze-properties.ts

```diff
@@ -152,6 +152,13 @@ export function analyzeProperties(
 
     prop.type = esutil.CLOSURE_CONSTRUCTOR_MAP.get(prop.type!) || prop.type;
 
+    if (prop.type && !typeTag) {
+      if (!nullableByDefault(prop.type)) {
```

Comment:
A comment here would be great - say that in Polymer we assume even non-nullable types to be nullable, or something.